src/webgl/loading.js

```diff
@@ -103,22 +103,42 @@ import './p5.Geometry';
  * @param  {String} [fileType]
  * @return {p5.Geometry} the <a href="#/p5.Geometry">p5.Geometry</a> object
  */
-p5.prototype.loadModel = function(path) {
+/**
+ * @method loadModel
+ * @param  {String} path
+ * @param  {Object} [options]
+ * @param  {function(p5.Geometry)} [options.successCallback]
+ * @param  {function(Event)} [options.failureCallback]
+ * @param  {String} [options.fileType]
```

Comment:
When I run `grunt yui:dev`, it looks like our reference UI unfortunately doesn't currently display these nested parameters:

<img width="1146" alt="image" src="https://github.com/processing/p5.js/assets/5315059/5391cfbf-64cc-47db-a998-007c9c6646dd">

[For the `createFramebuffer` docs](https://p5js.org/reference/#/p5/createFramebuffer) we list the options just in the description of the function as a bullet list:
https://github.com/processing/p5.js/blob/37d3324b457b177319ed65468201f2806a66eff5/src/core/rendering.js#L306-L313

Maybe we should do that here too?

Reply:
Sure, I just listed the options, do we need any kind of descriptions here?

Reply:
I think a short description would be helpful, maybe just a short sentence for each? Then we're good to go!

Reply:
Done!